pydatastructs/miscellaneous_data_structures/multiset.py

```diff
@@ -0,0 +1,41 @@
+__all__ = [
+    'Multiset'
+]
+
+
+class Multiset:
+    def __init__(self, *args):
+        self.counter = dict()
```

Comment:
Add a "TODO: Implement `dict` in `pydatastructs`".

Reply:
Done